playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
```

Comment:
Until recently, running `ansible-playbook playbooks/byo/config.yml --tags foo` would get you an early error on a role include that tried to use some fact that hadn't been set in a previous task (that was skipped due to not being tagged - it was actually a more complicated chain of events). That no longer breaks after the recent refactoring but still it seems unlikely to me that anyone is actually running playbooks with `--tags` because it did not work at all for a long time, and since we're not testing the use of `--tags` in CI it could easily stop working with any small change. So cargo-culting `tags: always` everywhere to me just seems misleading, if we can't certify that you can actually use the `--tags` feature with that playbook and don't have a known use case for it.